src/context/MessengerContext.js

```diff
@@ -89,6 +89,14 @@ class MessengerContext extends Context implements PlatformContext {
       return;
     }
 
+    if (this._event.isEcho || this._event.isDelivery || this._event.isRead) {
+      warning(
+        false,
+        'sendText: calling send APIs in echo, delivery or read events may cause endless self-responding, skipping...'
```

Comment:
How about

'sendText: calling send APIs to respond `message_reads`, `message_deliveries` or `message_echoes` events may cause endless self-responding, so it will be ignored by default. You should turn off subscription to those events or handle them correctly.'